packages/charts/src/charts/line/line-chart.component.tsx

```diff
@@ -54,6 +56,8 @@ export type Props = {
   areaMode?: boolean;
   /** Gradient on/off */
   gradient?: boolean;
+  /** Hover bar settings */
+  hoverBar?: HoverBarSettings;
```

Comment:
From my point of view the `hoverBar` should be a part of theme if we are going to add possibility to use multiple variants.

So rather than coupling it with `Line / Area` charts - I think it will be better to move it to theme - with possibility to enable / disable it.